src/utils/createStylingFromTheme.js

```diff
@@ -250,10 +257,11 @@ const getSheetFromColorMap = map => ({
 
   previewHeader: {
     padding: '5px 10px',
+    'flex-basis': '0px',
     'align-items': 'center',
     'border-bottom-width': '1px',
     'border-bottom-style': 'solid',
-    'min-height': '30px',
+    'min-height': '40px',
```

Comment:
Why was the height increased? I'm getting an unexpected extra space with that:
![image](https://cloud.githubusercontent.com/assets/7957859/19532935/690ac174-9646-11e6-8998-d44a11745046.png)

Which is fixed by changing to 
```diff
-flex: '0 0 40px',
+flex: '0 0 30px',
```